carmm/run/workflows/helper.py

```diff
@@ -0,0 +1,94 @@
+import os
+from fnmatch import fnmatch
+from ase.io import read
+
+class CalculationHelper:
+    def __init__(self,
+                 calc_type,
+                 parent_dir,
+                 filename,
+                 restart=True,
+                 verbose=True):
```

Comment:
You need to add comments to explain what these inputs are - this needs some comments added, please, otherwise it is opaque to readers.